google/cloud/pubsub/internal/default_batch_callback.h

```diff
@@ -50,6 +50,8 @@ class DefaultBatchCallback : public BatchCallback {
     message_callback_->user_callback(std::move(m));
   };
 
+  void EndMessage(std::string const&, std::string const&) override{};
```

Comment:
I suspect that ending semi-colon confuses `clang-format`.

```suggestion
  void EndMessage(std::string const&, std::string const&) override {}
```